contents/teams/replay/objectives.mdx

```diff
@@ -1,20 +1,28 @@
-# 📱Goal 1: People think of PostHog as a mobile solution
+# 🚀 Goal 1: Get Alex rocketing
 
-* onboard Yiannis and get him rocketing
-* mobile replay out of beta
-* flutter replay
+it's great to have Alex on the team... let's make it a success
 
-# 📹 Goal 2: Replay playback 4000
+# 👩‍🔬 Goal 2: Churn and retention exploration
 
-* Replay UX Overview
-    * we've invited @corywatilo to our November offsite where we will decide what this means for the last eighth of the year... 
-* replay playback performance
-    * we believe that if we improve playback we will see waiting time decrease, watch time increase, and number of recordings watched increase
-    * we believe that those metrics are strong proxies for customer satisfaction in replay
+We've had a lot of technology focus, let's have some product focus
 
-# 🎤 Goal 3: Ship the things users keep asking for
+This is a relatively vague goal, because we don't know enough to be specific. The goal is at a minimum to get to a place where we're operating from good knowledge
 
-* queryable network data
-* start replay on trigger e.g. event
+* find the user groups (e.g. low recording count vs high recording count)
+* talk to those users
+* find good metrics
+* run experiments to improve things
+* come out of this quarter either happy we can focus elsewhere or with a strong understanding of where to go next
 
-follow along in our [q4 megaissue](https://github.com/PostHog/posthog/issues/25061)
+# 🎤 Goal 3: Watch less and get more value
```

Comment:
from @raquelmsmith

> Vague feedback.... this doesn't feel very ambitious. I feel like the magic playlists stuff probably won't take more than a few weeks? Should we do the commenting/sharing bit?

interesting! so many interruptions this last quarter mean I put a brake on my self... i'm going to add something to cover that

but (without wanting to seem to ignore the feedback) this as well isn't a complete to-do list just the published goal. I'll add a mega-issue too